crates/config/src/unresolved_config.rs

```diff
@@ -27,6 +27,32 @@ use super::defaults::default_state_path;
 use crate::*;
 
 const DEFAULT_UTILITY_THREAD_ID: u32 = 1;
+const DEFAULT_HASHES_PER_ROUND: usize = 1024;
+const DEFAULT_ASYNC_TO_SYNC_QUEUE_SIZE: usize = 1;
+const DEFAULT_SYNC_TO_ASYNC_QUEUE_SIZE: usize = 1;
```

Comment:
what happens when this channel buffer is exhausted? Who will hold messages, and might it block some unexpected thread?

Reply:
It is the communication between a controlling Tokio task and a worker thread.  It may take little longer to react to the commands (stopping, submitting found proofs etc).  Tokio task is lightweight and should handle everything in time.